samples/llama_index_quick_start.ipynb

```diff
@@ -0,0 +1,1014 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "vyYQILyoEAqg"
+   },
+   "outputs": [],
+   "source": [
+    "from llama_index_alloydb_pg import AlloyDBChatStore, AlloyDBEngine\n",
+    "\n",
+    "engine = AlloyDBEngine.from_instance(\n",
+    "    project_id=project_id,\n",
+    "    instance=instance_name,\n",
+    "    region=region,\n",
+    "    cluster=cluster_name,\n",
+    "    database=database_name,\n",
+    "    user=\"postgres\",\n",
+    "    password=password,\n",
+    ")\n",
+    "chat_store_table_name = \"chat_store\"\n",
+    "\n",
+    "engine.init_chat_store_table(table_name=chat_store_table_name)\n",
+    "\n",
+    "chat_store = AlloyDBChatStore.create_sync(\n",
+    "    engine,\n",
+    "    table_name=chat_store_table_name,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a ChatMemoryBuffer\n",
+    "The `ChatMemoryBuffer` stores a history of recent chat messages, enabling the LLM to access relevant context from prior interactions.\n",
+    "\n",
+    "By passing our chat store into the `ChatMemoryBuffer`, it can automatically retrieve and update messages associated with a specific session ID or `chat_store_key`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from llama_index.core.memory import ChatMemoryBuffer\n",
+    "\n",
+    "memory = ChatMemoryBuffer.from_defaults(\n",
+    "    token_limit=3000,\n",
+    "    chat_store=chat_store,\n",
+    "    chat_store_key=\"user1\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create a LlamaIndex `ChatEngine`\n",
+    "\n",
+    "You can re-use the `VectorStoreIndex` created above to create a [ChatEngine](https://docs.llamaindex.ai/en/stable/module_guides/deploying/chat_engines/) which includes a `ChatStore` to save the chats between the user and AI assistant."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "2yuXYLTCl2K1"
+   },
+   "source": [
+    "Here is an example of how you would add a chat message and fetch all messages from the chat store."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "qDVoTWZal0ZF"
+   },
+   "outputs": [],
+   "source": [
+    "chat_engine = index.as_chat_engine(chat_mode=\"context\", memory=memory)\n",
+    "\n",
+    "op = chat_engine.chat(\"What was the cast in Blood and Water?\")\n",
+    "op = chat_engine.chat(\"How many seasons are there for Kota Factory?\")\n",
+    "# Retrieve response\n",
+    "op.response\n",
```

Comment:
remove line since we are printing responses below

Reply:
actually both are used for a different purpose. 
`.chat` gets message response with prompt + chat history
`.get_messages` gets all messages for a given key.

Adding clarification for that.

Reply:
Sorry, this was suppose to mean - do we need "op.response" if we are printing the messages below? Do we want to `print(op.response)`?